package.json

```diff
@@ -10,7 +10,7 @@
     "ecs-task-runner": "./bin/ecs-task-runner"
   },
   "scripts": {
-    "test": "NODE_ENV=test ./node_modules/.bin/mocha --recursive"
+    "test": "NODE_ENV=test ./node_modules/.bin/mocha --parallel --recursive"
```

Comment:
would avoid parallel usage, I remember some problems with one of these codebases where it was not really concurrency safe

Reply:
Well, this is one of them apparently. I removed parallel and now tests won't pass lol

Reply:
https://github.com/m-radzikowski/aws-sdk-client-mock/issues/64
Apparently mocha has weird behavior with aws-sdk-client-mock, shuffling up the mockClient invocations resolves it.